### ISSUE TITLE
Give members in discord a role that reflects their role in the DAO

### ISSUE THREAD

**Issue by july5613** [FantomMenaceDAO/MIPs#4]
# Menace Improvement Proposal (MIP-9)

## tl;dr
Give DAO members roles in discord that reflect our role in the DAO

## What would be the roles?
- Software 
- Governance
- Artists/Creators
- BizDev
- Marketing/Social/Community
- Design/Frontend
- Investor/fund management/back of the office/Bookkeeping

Please comment below if you think that there should be more roles

## Motivation
We currently don't have a good idea of who does what in the DAO, it would be nice to have explicit roles, that maybe in the future when you get onboarded you can choose your role, and it'll assign you to guilds.

## Owners
List folks who are going to **owners** for the project and be responsible for delivering
July, and anyone else who's interested

## Resources
Do you need funding? Do the contributors to making this improvement need to be paid in NFTs or FTM?
- Nope
